dkms.in

```diff
@@ -1336,17 +1315,16 @@ do_install()
                     debian* | ubuntu* )
                         ;;
                     *)
-                       echo $"     - Stored $module_dup"
-                       mkdir -p "$dkms_tree/$module/original_module/$kernelver/$arch/collisions/$dup_tree"
-                       mv -f $module_dup "$dkms_tree/$module/original_module/$kernelver/$arch/collisions/$dup_tree"
-                       ;;
+                        echo $"All instances of this module will now be stored for reference purposes ONLY in $dkms_tree/$module/original_module/$kernelver/$arch/collisions/"
```

Comment:
I was thinking more of printing this for every module in the case of multiple kernel modules (just some out of all the modules in the `dkms.conf` file could be duplicate). The state as it is now, it prints only the first of the various modules included in the DKMS "package", instead of all of then.

I think I will change it to be more generic and get it back out of the loop, being too detailed does not help.

Reply:
As long as the information about the `collisions` folder is there, it should be enough.